src/frontend/shared-components/src/components/table/Table.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./Table.css";
+
+const TableElement = ({ element: { name, value } }) => {
+  return (
+    <tr>
+      <td>{name}</td>
+      <td>{value}</td>
+    </tr>
+  );
+};
+
+export default function Table({
+  table: { id, header, tableElements, tableStyle }
+}) {
+  const tableComponents = tableElements.map(element => {
+    return <TableElement key={element.key || element.name} element={element} />;
+  });
+
+  return (
+    <table id={id} className={tableStyle}>
+      <thead>
+        <tr>
+          <th colSpan="2">{header}</th>
+        </tr>
+      </thead>
+      <tbody>{tableComponents}</tbody>
```

Comment:
dumb formatting question:

tbody>
   { blah {
</tbody

vs

tbody> { blah } </tbody

whats better?

Reply:
prettier will handle the formatting for us so it should decide what it thinks is best :) (based on line length, etc)

Reply:
I think if there were multiple things in \<tbody\> then separate line?